samples/drivers/led_strip/README.rst

```diff
@@ -44,11 +44,11 @@ property. See the overlay file
 :zephyr_file:`samples/drivers/led_ws2812/boards/thingy52_nrf52832.overlay` for more detail.
```

Comment:
There are still several occurrences of led_ws2812 to replace with led_string (such above) in this file ^

Reply:
All `led_ws2812` change to `led_strip`.